share/instruments/update_instruments_xml.py

```diff
@@ -360,83 +360,87 @@ def disambiguation(instrumentId: str, nameType: str):
+ */
+
+""")
 
     # Templates
+    # TODO: generate based on categories.json
     f.write("// Templates\n")
     d = "../templates"
     # sort to get same ordering on all platforms
     for o in sorted(os.listdir(d)):  
         ofullPath = os.path.join(d, o)
         if os.path.isdir(ofullPath):
             templateCategory = o.split("-")[1].replace("_", " ")
-            add_translatable_string(f, 'Templates', templateCategory)
+            add_translatable_string(f, 'project/templatecategory', templateCategory)
 
             # sort to get same ordering on all platforms
             for t in sorted(os.listdir(ofullPath)):
                 if os.path.isdir(os.path.join(ofullPath, t)):
                     templateName = t.split("-")[1].replace("_", " ")
-                    add_translatable_string(f, 'Templates', templateName)
+                    add_translatable_string(f, 'project/template', templateName)
 
     f.write("\n")
     f.write("// Genres\n")
     for genre in genres.values():
-        add_translatable_string(f, 'InstrumentsXML', genre['name'])
+        add_translatable_string(f, 'engraving/instruments/genre', genre['name'])
 
     f.write("\n")
     f.write("// Families\n")
     for family in families.values():
-        add_translatable_string(f, 'InstrumentsXML', family['name'])
+        add_translatable_string(f, 'engraving/instruments/family', family['name'])
 
     f.write("\n")
     f.write("// Groups & Instruments\n")
     for group in groups.values():
         f.write("\n// " + group['name'] + "\n")
-        add_translatable_string(f, 'InstrumentsXML', group['name'])
+        add_translatable_string(f, 'engraving/instruments/group', group['name'])
 
         for instrument in instruments[group['id']].values():
             f.write('\n')
             instrumentId = instrument['id']
 
-            add_translatable_string_if_not_null(f, 'InstrumentsXML', instrument['description'],
+            add_translatable_string_if_not_null(f, 'engraving/instruments', instrument['description'],
                                                 disambiguation(instrumentId, 'description'))
 
             for nameType in ['trackName', 'longName', 'shortName']:
-                add_translatable_string_if_not_null(f, 'InstrumentsXML', instrument[nameType],
+                add_translatable_string_if_not_null(f, 'engraving/instruments', instrument[nameType],
                                                     disambiguation(instrumentId, nameType))
                 
             if instrument['traitName'] != '[hide]':
-                add_translatable_string_if_not_null(f, 'InstrumentsXML', instrument['traitName'],
+                add_translatable_string_if_not_null(f, 'engraving/instruments', instrument['traitName'],
                                                     disambiguation(instrumentId, 'traitName'), 
                                                     'traitName codes: * most common for instrument; () hide in score')
```

Comment:
@shoogle Considering your explanation about the meaning of `*`, shouldn't we update/extend this comment? Maybe we should put a link to some external documentation here, since a decent explanation of the possibilities translators have would maybe not fit in a comment.

Reply:
You're right. I'll put something together quickly so you have a URL to include.

The comment could also mention the traitType for the instrument (transposition, tuning or course).